docs/src/user-guide/quick-start/setup/main-page.md

```
@@ -0,0 +1,131 @@
+
+To run properly on your system, CLP requires a few other programs. Before you set up CLP, ensure that you have the following installed.
+
+### Docker
+
+CLP uses Docker to deploy its different components. You can check whether Docker is already installed on your system by running the command
+
+```bash
+docker version
+```
+
+If Docker is not on your system, follow the instructions [here][Docker] to install it. 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)